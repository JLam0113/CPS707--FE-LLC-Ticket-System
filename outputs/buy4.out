Welcome, please enter a command.
Please enter your username
Non-privileged Login successful, now logged in as user1, please enter a command
Buy tickets selected, please enter the event title:
Please enter the number of tickets:
