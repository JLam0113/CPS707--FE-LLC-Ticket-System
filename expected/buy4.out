Welcome, please enter a command.
Please enter your username
Non-privileged Login successful, now logged in as user1, please enter a command
Buy tickets selected, please enter the event title:
Please enter the number of tickets:
Please enter the seller's username:
You are buying 4 tickets for event2 from admin for $10, can you please confirm with yes/no?
Transaction confirmed, please enter a command.
Buy tickets selected, please enter the event title:
Please enter the number of tickets:
Please enter the seller's username:
Only 4 tickets can be purhcased per session. Please exit and return to purchase more, please enter a command.
Session ended
